radar-auth/src/main/java/org/radarcns/auth/authentication/TokenValidator.java

```diff
@@ -116,12 +112,14 @@ public RadarToken validateAccessToken(String token) throws TokenValidationExcept
         for (JWTVerifier verifier : getVerifiers()) {
             try {
                 DecodedJWT jwt = verifier.verify(token);
+                log.debug("JWT claims from token {} are {}", token, jwt.getClaims());
                 Set<String> claims = jwt.getClaims().keySet();
-                Set<String> missing = REQUIRED_CLAIMS.stream()
-                        .filter(c -> !claims.contains(c)).collect(Collectors.toSet());
-                if (!missing.isEmpty()) {
-                    throw new TokenValidationException("The following required claims were "
-                            + "missing from the token: " + String.join(", ", missing));
+                log.debug("Claims included in the token are {}", claims);
+
+                // check for scope claim
+                if (!claims.contains(JwtRadarToken.SCOPE_CLAIM)) {
```

Comment:
Perhaps shorten/reword this a bit to
```java
Map<String, Claim> claims = jwt.getClaims();
log.debug("JWT claims from token {} are {}", token, claims);
 if (!claims.containsKey(JwtRadarToken.SCOPE_CLAIM)) {
 throw new TokenValidationException("The required claim " + JwtRadarToken.SCOPE_CLAIM + " is"
                         + "missing from the token.");
```
For single debug line and exception readability.